bundle/manifests/bpfman-operator.clusterserviceversion.yaml

```diff
@@ -268,22 +284,20 @@ spec:
   description: "The bpfman Operator is a Kubernetes Operator for deploying [bpfman](https://bpfman.netlify.app/),
     a system daemon\nfor managing eBPF programs. It deploys bpfman itself along with
     CRDs to make deploying\neBPF programs in Kubernetes much easier.\n\n## Quick Start\n\nTo
-    get bpfman up and running quickly with self signed certificates simply do \nthe
-    following.\n\n```bash\nkubectl create namespace openshift-bpfman\n```\n After
-    creating the namespace simply install the bpfman-operator in the desired namespace
-    from operator-hub\n## Configuration\n\nThe `bpfman-config` configmap is automatically
-    created in the `bpfman` namespace and used to configure the bpfman deployment.\n\nTo
-    edit the config simply run\n\n```bash\nkubectl edit cm bpfman-config\n```\n\nThe
-    following fields are adjustable\n\n- `bpfman.agent.image`: The image used for
-    the bpfman-agent, defaults to `quay.io/bpfman/bpfman-agent:latest`\n- `bpfman.image`:
-    The image used for bpfman, defaults to `quay.io/bpfman/bpfman:latest`\n- `bpfman.log.level`:
-    the log level for bpfman, currently supports `debug`, `info`, `warn`, `error`,
-    and `fatal`, defaults to `info`\n- `bpfman.agent.log.level`: the log level for
-    the bpfman-agent currently supports `info`, `debug`, and `trace` \n\nThe bpfman
-    operator deploys eBPF programs via CRDs. The following CRDs are currently available,
-    \n\n- XdpProgram\n- TcProgram\n- TracepointProgram\n- KprobeProgram\n- UprobeProgram\n-
-    FentryProgram\n- FexitProgram\n\n## More information\n\nPlease checkout the [bpfman
-    community website](https://bpfman.io/) for more information."
+    get bpfman up and running quickly simply click 'install' to deploy the bpfman-operator
+    in the bpfman namespace via operator-hub.\n## Configuration\n\nThe `bpfman-config`
+    configmap is automatically created in the `bpfman` namespace and used to configure
+    the bpfman deployment.\n\nTo edit the config simply run\n\n```bash\nkubectl edit
```

Comment:
Is the bundle more OCP specific? If so, should `kubectl` be replaced with `oc`?

Reply:
So technically the operatorhub is for everyone, and community operators prod is for OCP I think we can leave it as kubectl here and add a carry commit to  https://github.com/redhat-openshift-ecosystem/community-operators-prod